### PR TITLE
Find Forms user testing for two different alert boxes

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
We are testing with users two different alert boxes. 
Both boxes behind a feature flag

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/27426
https://github.com/department-of-veterans-affairs/va.gov-team/issues/27424

## Testing done
Updated Unit tests, ran unit, integration, and e2e. 
Spun up locally 

## Screenshots
Screen Shot 2021-07-16 at 5 26 40 PM
Screen Shot 2021-07-16 at 5 22 48 PM
Screen Shot 2021-07-16 at 5 16 04 PM

## Acceptance criteria
- [x] Things run right...

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/find-forms/components/SearchResult.jsx [hunk 1]
```diff
@@ -181,19 +189,19 @@ const SearchResult = ({ form, formMetaInfo, showPDFInfoBox }) => {
...
-                You'll need to download this form and open it in Adobe Acrobat
-                Reader
-              </h3>
-              <a href="https://www.va.gov/resources/what-if-im-having-trouble-opening-a-pdf/">
-                Get instructions for opening the form in Acrobat Reader
-              </a>
-            </div>
-          </va-alert>
+      {showPDFInfoVersionOne ? (
+        <dd className="find-forms-alert-message vads-u-margin-bottom--2 vads-u-background-color--primary-alt-lightest">
```
Comment (zacharymorel) [inline src/applications/find-forms/components/SearchResult.jsx:193]: Note, this will be apart of the experimental design system.

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/find-forms/components/FormTitle.jsx [hunk 2]
```diff
@@ -30,10 +56,12 @@ const FormTitle = ({ id, formUrl, title, lang, recordGAEvent }) => (
 );
 
 FormTitle.propTypes = {
+  currentPosition: PropTypes.number,
   id: PropTypes.string.isRequired,
   formUrl: PropTypes.string,
   recordGAEvent: PropTypes.func,
   title: PropTypes.string.isRequired,
```
Comment (kelsonic) [inline src/applications/find-forms/components/FormTitle.jsx:63]: I think we're missing `lang` here

File: src/applications/find-forms/containers/SearchResults.jsx [hunk 3]
```diff
@@ -240,7 +244,7 @@ SearchResults.propTypes = {
   hasOnlyRetiredForms: PropTypes.bool.isRequired,
   sortByPropertyName: PropTypes.string,
   startIndex: PropTypes.number.isRequired,
-  showPDFInfoBox: PropTypes.bool,
+  showPDFInfoVersionOne: PropTypes.bool,
```
Comment (kelsonic) [inline src/applications/find-forms/containers/SearchResults.jsx:247]: Should we add `showPDFInfoVersionTwo` too? 🙂

File: src/applications/find-forms/sass/find-va-forms.scss [hunk 4]
```diff
@@ -34,6 +34,23 @@
     border-color: $color-gray-lighter;
     border-style: solid;
   }
+
+  .find-forms-alert-message {
+    display: flex;
+    padding: 3.2rem 6.4rem 3.2rem 2.4rem;
+    width: 100%;
```
Comment (kelsonic) [inline src/applications/find-forms/sass/find-va-forms.scss:41]: The 3 styles above can be written as `vads-u-display--flex vads-u-padding-y--3 vads-u-padding-x--7 vads-u-width--full` or something close to that if you want 🙂

File: src/applications/find-forms/tests/components/SearchResult.unit.spec.jsx [hunk 5]
```diff
@@ -92,13 +92,33 @@ describe('Find VA Forms <SearchResult />', () => {
     tree.unmount();
   });
 
-  it('should have have a Alert Box PDF information for download trouble', () => {
+  it('should have have a Alert Box PDF information (version one) for download trouble', () => {
```
Comment (Rleahy22) [inline src/applications/find-forms/tests/components/SearchResult.unit.spec.jsx:95]: nit: -have

File: src/applications/find-forms/sass/find-va-forms.scss (hunk 4 above)
Comment (zacharymorel) [reply src/applications/find-forms/sass/find-va-forms.scss:41]: Awesome, thank you!

**Review by Rleahy22 — APPROVED** [review]

File: src/applications/find-forms/components/FormTitle.jsx (hunk 2 above)
Comment (zacharymorel) [reply src/applications/find-forms/components/FormTitle.jsx:63]: Thank you

File: src/applications/disability-benefits/686c-674/config/helpers.js [hunk 6]
```diff
@@ -34,7 +34,7 @@ export const VaFileNumberMissingAlert = (
       number.
     </p>
     <p className="vads-u-font-size--base">
-      You'll need to update your personal information. Please call Veterans
```
Comment (zacharymorel) [inline src/applications/disability-benefits/686c-674/config/helpers.js:37]: Looks like this was from the new eslint rule?

File: src/applications/check-in/pages/Confirmation.jsx [hunk 7]
```diff
@@ -21,8 +21,8 @@ const Confirmation = ({ appointment }) => {
           You’re checked in
         </h1>
         <p>
-          Please wait in the {clinicName} waiting room. We'll come get you when
-          it's time for your appointment to start.
```
Comment (zacharymorel) [inline src/applications/check-in/pages/Confirmation.jsx:25]: Looks like this was from the new eslint rule?

Comment (Mottie) [reply src/applications/check-in/pages/Confirmation.jsx:25]: yep

**Review by mdewey — APPROVED** [review]
LGTM.

File: src/applications/find-forms/components/SearchResult.jsx [hunk 8]
```diff
@@ -181,19 +189,19 @@ const SearchResult = ({ form, formMetaInfo, showPDFInfoBox }) => {
...
-              </h3>
-              <a href="https://www.va.gov/resources/what-if-im-having-trouble-opening-a-pdf/">
-                Get instructions for opening the form in Acrobat Reader
-              </a>
-            </div>
-          </va-alert>
+      {showPDFInfoVersionOne ? (
+        <dd className="find-forms-alert-message vads-u-margin-bottom--2 vads-u-background-color--primary-alt-lightest vads-u-display--flex vads-u-padding-y--4 vads-u-padding-right--7 vads-u-padding-left--3 vads-u-width--full">
+          <i aria-hidden="true" role="img" />
+          <span className="sr-only">Alert: </span>
```
Comment (Mottie) [inline src/applications/find-forms/components/SearchResult.jsx:195]: From the screenshot, the `<i />` above is an info icon? So should this say "Alert"?

File: src/applications/disability-benefits/686c-674/config/helpers.js (hunk 6 above)
Comment (micahchiang) [reply src/applications/disability-benefits/686c-674/config/helpers.js:37]: Sorry that this bled into your PR!

File: src/applications/check-in/pages/Confirmation.jsx (hunk 7 above)
Comment (Mottie) [reply src/applications/check-in/pages/Confirmation.jsx:25]: Weird that it is automatically fixing some lines, but not others...

**Review by micahchiang — APPROVED** [review]

File: src/applications/find-forms/components/SearchResult.jsx (hunk 8 above)
Comment (zacharymorel) [reply src/applications/find-forms/components/SearchResult.jsx:195]: Yes!

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**